index.js

```diff
@@ -25,46 +10,65 @@ function loadBooks(bookCollection) {
 		bookCollection.forEach((book, i) => {
 			sectionBooks.insertAdjacentHTML(
 				'afterbegin',
-				returnBook(book.title, book.author, i),
+				books.returnBook(book.title, book.author, i)
 			);
 		});
 	}
 	/* eslint-enable */
 }
 
-function addBook(title, author) {
-  sectionBooks.insertAdjacentHTML('afterbegin', returnBook(title, author));
+class BookApp {
+  constructor() {
+    this.bookCollection = localStorage.getItem('bookCollection')
+      ? JSON.parse(localStorage.getItem('bookCollection'))
+      : [];
+  }
+
+  addBook(title, author) {
+    this.bookCollection.push({
+      title,
+      author,
+    });
```

Comment:
- [ ] **GOOD JOB 🔥👌**. You have implemented js classes successfully. I have suggestion for you. Instead of array `push` method I would recommend using array `concat` method 🍀